vulcan/lib/server/controllers/data_controller.rb

```diff
@@ -11,9 +11,17 @@ def fetch
 
     raise Etna::NotFound, "No data for value #{@params[:data]}." unless ["steps", "pools"].include?(@params[:data])
 
+    case @params[:data]
```

Comment:
Note, I basically expect to throw all this controller out, after we hook into Archimedes.